lucet-runtime/lucet-runtime-internals/src/instance/execution.rs

```diff
@@ -244,6 +244,11 @@ pub unsafe extern "C" fn exit_guest_region(instance: *mut Instance) {
                 // We finished executing the code in our guest region normally! We should reset
                 // the kill state, invalidating any existing killswitches' weak references.
                 mem::drop(current_domain);
+                // There should be only one strong reference to `kill_state`, so as a consistency
+                // check ensure that's still true. This is necessary so that when we drop this
+                // `Arc`, weak refs are no longer valid. If this assert fails, something cloned
+                // `KillState`, or a `KillSwitch` has upgraded it's ref - both of these are errors!
```

Comment:
```suggestion
                // `KillState`, or a `KillSwitch` has upgraded its ref - both of these are errors!
```

I appreciate the explanatory comments here. :heart: